relation_engine_server/utils/spec_loader.py

```diff
@@ -74,11 +74,18 @@ def get_schema(schema_type, name, path_only=False):
 
     schema_search_type = pluralise_schema_type(schema_type)
 
-    try:
-        path = _find_paths(_CONF['spec_paths'][schema_search_type], name + '.yaml')[0]
-    except IndexError:
+    yaml_paths = _find_paths(_CONF['spec_paths'][schema_search_type], f'{name}.yaml')
+    json_paths = _find_paths(_CONF['spec_paths'][schema_search_type], f'{name}.json')
+    # ensure we're using the canonical path and that all paths are unique
+    # we are only interested in paths that are in the designated spec repo
+    repo_path = os.path.abspath(_CONF['spec_paths']['repo'])
+    all_paths = [p for p in set(os.path.abspath(path) for path in yaml_paths + json_paths) if repo_path in p]
```

Comment:
Using `p.startswith(repo_path)` I think would be more accurate

I would avoid a nested list comprehension like this to help with readability

Reply:
OK, have used `p.startswith` and split out the list comprehension. 👍